Northstar.Client/mod/scripts/vscripts/ui/panel_mainmenu.nut

```diff
@@ -571,12 +571,12 @@ void function TryAuthWithLocalServer()
 	{
 		CloseAllDialogs()
 
-		var reason = NSGetAuthFailReason()
+		string reason = NSGetAuthFailReason()
 
 		DialogData dialogData
 		dialogData.image = $"ui/menu/common/dialog_error"
 		dialogData.header = "#ERROR"
-		dialogData.message = Localize("#NS_SERVERBROWSER_CONNECTIONFAILED") + "\nERROR: " + reason  + "\n" + Localize("#" + reason)
+		dialogData.message = reason
```

Comment:
I like no localization more tbh

Reply:
It's objectively worse for users. All helpers need is the error code so it doesn't affect them (except all our docs and stuff were based around the error code, which is no longer shown to the user at all)

Reply:
We definitely should at the very least include the unlocalized error code.

We could also do something like this:

```
Connection failed! // Localized message
Code: <code> // Unlocalized code

Check 'wiki.northstar.tf/error_codes' for possible solutions // Localized message
```

Reply:
My current thoughts is that we should move the "Connection failed!" bit to the header of the dialog as well.
Currently it looks like
```
ERROR // header

Couldn't find player account
```
Before it looked like
```
ERROR // header

Connection failed! // localised
ERROR: PLAYER_NOT_FOUND
Couldn't find player account // localised
```

IMO, it should look more like
```
Connection Failed! // header

Couldn't find player account // localised
Error code: PLAYER_NOT_FOUND

Check 'wiki.northstar.tf/error_codes' for possible solutions // localised
```

Reply:
![image](https://github.com/R2Northstar/NorthstarMods/assets/66967891/23e83fce-a6da-49f5-bdcf-2303efabf30c)
Something like this could be good, since this isnt a disconnect dialog we can add more buttons and do stuff like opening links. Could be good to get fewer people asking for help in discord?